src/widget/OktaSignIn.js

```diff
@@ -117,9 +116,17 @@ var OktaSignIn = (function () {
     require.config(requireConfig);
     OktaAuth = require('vendor/OktaAuth');
     Util = require('util/Util');
+    LoginRouter = require('LoginRouter');
+
 
     authClient = new OktaAuth({ uri: options.baseUrl, transformErrorXHR: Util.transformErrorXHR });
-    _.extend(this, getProperties(authClient, options));
+    _.extend(this, getProperties(authClient, options, LoginRouter));
```

Comment:
nit: personal preference, but I think it'd be nicer to have LoginRouter as the first or second arg (i.e. options usually go last).